tasks/vscode.py

```diff
@@ -0,0 +1,58 @@
+"""
+vscode namespaced tags
+
+Helpers for getting vscode set up nicely
+"""
+import json
+import os
+from collections import OrderedDict
+
+from invoke import task
+
+from .build_tags import build_tags, filter_incompatible_tags, get_build_tags, get_default_build_tags
+from .flavor import AgentFlavor
+
+VSCODE_DIR = ".vscode"
+VSCODE_FILE = "settings.json"
+
+
+@task
+def set_buildtags(
+    _,
+    target="agent",
+    build_include=None,
+    build_exclude=None,
+    flavor=AgentFlavor.base.name,
+    arch='x64',
+):
+    """
+    Modifies vscode settings file for this project to include correct build tags
+    """
+    flavor = AgentFlavor[flavor]
+
+    if target not in build_tags[flavor].keys():
+        print("Must choose a valid target.  Valid targets are: \n")
+        print(f'{", ".join(build_tags[flavor].keys())} \n')
+        return
+
+    build_include = (
+        get_default_build_tags(build=target, arch=arch, flavor=flavor)
+        if build_include is None
+        else filter_incompatible_tags(build_include.split(","), arch=arch)
+    )
+    build_exclude = [] if build_exclude is None else build_exclude.split(",")
+    use_tags = get_build_tags(build_include, build_exclude)
```

Comment:
In `build_tags.py`, there is a `compute_build_tags_for_flavor ` function that does this: https://github.com/DataDog/datadog-agent/blob/c57fc32876cc3f71b0c9fece4b86f06419c4a7c5/tasks/build_tags.py#L178-L197 Could you use it?